src/WebJobs.Script/HttpProxyService/HttpForwardingException.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.Azure.WebJobs.Script.Grpc.Exceptions
+namespace Microsoft.Azure.WebJobs.Script
```

Comment:
Why is the namespace changing for all these files?

Reply:
These files were all moved to the `WebJobs.Script` project (they were previously in the `Grpc` project).

Reply:
Are they common items that will be used by others? Shouldn't this move to `src/WebJobs.Script/Exceptions/HttpForwardingException.cs`  and vice versa for the rest? If not, should you add `HttpProxyService` to namespace? i.e. `namespace Microsoft.Azure.WebJobs.Script.HttpProxyService`